### ISSUE TITLE
[Question/Feature] how set converter globally and override builtin

### ISSUE THREAD

**Issue by gemunet** [google/json_serializable.dart#1272]
How can I set a converter globally to replace the builtin datetime for example?

this does not appear in the documentation

https://pub.dev/packages/json_serializable

replace the datetime converter with a custom one globally for all classes

**Comment by Ali1Ammar** [comment]
i also need this feature to convert all datetime to local time

**Comment by kevmoo** [comment]
there is not an easy way to do this, sadly. At least not yet. Will flag this as a feature request.

**Comment by Nico04** [comment]
> > there is not an easy way to do this, sadly. At least not yet. Will flag this as a feature request.
> 
> Don't converters solve this problem already? For example:
> 
> ```dart
> class MyJsonSerializable extends JsonSerializable {
> 	const MyJsonSerializable({
> 		super.fieldRename,
> 		// etc., forward all props
> 	}) : super(converters: [MyGlobalConverter()]);
> }
> ```
> 
> Then just always use `@MyJsonSerializable` instead of `@JsonSerializable`

This is all right, but it would be even better if it was just like the in-built types like DateTime : you have nothing to specify.
So for custom class on a project, you would just have to specify converter once globally, and nowhere else. That would be very handy !
